### PR TITLE
fix: support Koa2 / Node 6.2

### LINKED ISSUE #18: Not setting headers on Koa2 / Node 6.2
The headers don't seem to be getting set on the latest version of Koa 2 with node 6.2. Any ideas?

### PR DESCRIPTION
**Pull request by leecade**
fixes #18

### CODE REVIEW

**Comment by fengmk2** [comment]
@leecade ci test fail

File: index.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ module.exports = function(options) {
   return function cors(ctx, next) {
     // If the Origin header is not present terminate this set of steps.
     // The request is outside the scope of this specification.
-    const requestOrigin = ctx.get('Origin');
+    const requestOrigin = ctx.get('Origin') || ctx.origin;
```
Comment (fengmk2) [inline index.js]: can you add a test case for this fix?

**Comment by arg20** [comment]
can this be rebased and merged?

**Comment by RobertHerhold** [comment]
What's the status of this? @fengmk2

**Comment by fengmk2** [comment]
@leecade should rebase and add a test case for this change.

File: index.js (hunk 1 above)
Comment (cosmosgenius) [inline index.js]: ctx.origin points to the url to which the client connects to not the origin header right? Is this fix correct?

**Comment by luisfarzati** [comment]
I think this PR can be closed since Node 6 support was just dropped since 3.0.0.

**Comment by dead-horse** [comment]
close for now, seems it is not a fix for this problem. :(

**Pull request closed without merging**